webapp/src/ts/modules/tasks/tasks.component.ts

```diff
@@ -164,4 +198,14 @@ export class TasksComponent implements OnInit, OnDestroy {
   listTrackBy(index, task) {
     return task?._id;
   }
+
+  async getCurrentLineageLevel(){
```

Comment:
No `await`, no need of `async`
```suggestion
  getCurrentLineageLevel(){
```